index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
+    <title>Apple</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <ul>
+          <li class="li-logo">
+            <a href=""><img src="img/logo.png" class="logo" alt="" /> </a>
+          </li>
+          <li>
+            <a href=""><span>Store</span></a>
+          </li>
+          <li>
+            <a href=""><span>Mac</span></a>
+          </li>
+          <li>
+            <a href=""><span>IPod</span></a>
+          </li>
+          <li>
+            <a href=""><span>IPone</span></a>
+          </li>
+          <li>
+            <a href=""><span>IPad</span></a>
+          </li>
+          <li class="li-itunes">
+            <a href=""><span>Itunes</span></a>
+          </li>
+          <li class="li-support">
+            <a href=""><span>Support</span></a>
+          </li>
+          <li>
+            <input class="nav-input" type="text" placeholder="" />
+          </li>
+        </ul>
+      </nav>
+
+      <div class="hero-section">
+        <a href="">
+          <img
+            src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png"
```

Comment:
Image should be properly displayed, please adjust it, there are some parts on the bottom that shouldn't be seen.